### ISSUE TITLE
error - brick

### ISSUE THREAD

**Issue by eiiiiix** [Voxel9/Testkit-Installer-Deluxe#1]
sometimes (rarely) when installing testkit fw i get the C- (generic common error) and the app closes before it finishes
then if i go to restore retail or finish testkit the installer gives me the error when i launch it
on reboot it softbricks
reinstalling the firmware fixes it but its a minor annoyance when it happens

**Comment by Voxel9** [comment]
Been a while since I had anything to do with this, but thanks; I'll investigate this sometime.

**Comment by dothackjhe** [comment]
Will this be updated for 3.65 Enso?

**Comment by MobCat** [comment]
Yeah There was a TestKit installer for 3.65 (https://github.com/SilicaAndPina/CEX-2-REX/releases)
Dont install this on 3.65 Enso and knowing that befoul hand would of been nice... think i just bricked my console :P
